INSTALL.md

```diff
@@ -55,19 +54,19 @@ Install Leaflet and the Leaflet extension that provides the edit link:
 $ make install-deps
 ```
 
- Install osm2pgsql 0.96.0 or greater (Ubuntu and Debian users may use their `osm2pgsql` package provided by their distribution instead, if it's new enough):
+ Install osm2pgsql 0.96.0 or greater:
 
 ```shell
 $ cd import
-$ wget https://github.com/openstreetmap/osm2pgsql/archive/0.96.0.tar.gz
-$ tar -xvzf 0.96.0.tar.gz
+$ wget https://github.com/openstreetmap/osm2pgsql/archive/1.9.2.tar.gz
+$ tar -xvzf 1.9.2.tar.gz
 $ mkdir osm2pgsql-build
 $ cd osm2pgsql-build
-$ cmake ../osm2pgsql-0.96.0
+$ cmake ../osm2pgsql-1.9.2
 $ make
 $ sudo make install
 $ cd ..
-$ rm -fr osm2pgsql-build osm2pgsql-0.96.0 0.96.0.tar.gz
+$ rm -fr osm2pgsql-build osm2pgsql-1.9.2 1.9.2.tar.gz
```

Comment:
I would omit the `rm` here as well as in other places. Everyone should be able to clean that up alone if it is relevant.